systemtest/src/test/java/io/strimzi/systemtest/RollingUpdateST.java

```diff
@@ -229,15 +232,36 @@ void testKafkaAndZookeeperScaleUpScaleDown() throws Exception {
         LOGGER.info("Scaling up to {}", scaleTo);
         // Create snapshot of current cluster
         String kafkaStsName = kafkaStatefulSetName(CLUSTER_NAME);
-        KafkaResource.replaceKafkaResource(CLUSTER_NAME, k -> k.getSpec().getKafka().setReplicas(scaleTo));
+
+        KafkaResource.replaceKafkaResource(CLUSTER_NAME, kafka -> {
+            kafka.getSpec().getKafka().setReplicas(scaleTo);
+
+            JbodStorage jbodStorage =  new JbodStorageBuilder()
```

Comment:
Why do you use JBOD instead of classic persistent storage?

Reply:
So as far as i know, we do not have a persistent storage. We have only persistent claim storage and with that JBOD can consist list of `EphemeralStorage` or `PersistentClaimStorage`  = https://strimzi.io/docs/master/#type-PersistentClaimStorageOverride-reference. But i am not sure about it. Does it make sense?

Reply:
Using Jbod alone would be ok. But I suspect you are using it wrongly. Please see my comment.